act-rules-format.bs

```diff
@@ -15,21 +15,21 @@ Markup Shorthands: markdown yes
 Introduction {#intro}
 =====================
 
-There are currently many tools available which aid their users in testing web content for conformance to accessibility standards such as the Web Content Accessibility Guidelines (WCAG). As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
+There are currently many tools available which aid their users in testing web content for conformance to accessibility standards such as the [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/) [[WCAG20]]. As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
 
-This format is intended to enable a consistent interpretation of how to test for accessibility requirements to avoid conflicting results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools (ATTs).
+This format is intended to enable a consistent interpretation of how to test for [accessibility requirements](#structure-accessibility-requirements) to avoid conflicting results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools (ATTs).
 
 Describing how to test certain accessibility requirements will result in accessibility tests that are transparent with test results that are reproducible. The Accessibility Conformance Testing (ACT) Rules Format defines the requirements for these test descriptions, known as Accessibility Conformance Testing Rules (ACT Rules).
 
 
 Scope {#scope}
 ==============
 
-The ACT Rules Format defined in this specification is focused on the description of rules applicable to content created using web technologies, such as HTML, CSS, WAI-ARIA, SVG and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning that it can conceivably be used to describe test rules for other technologies.
+The ACT Rules Format defined in this specification is focused on the description of rules applicable to content created using web technologies, such as [Hypertext Markup Language](https://www.w3.org/TR/html5/) [[HTML5]], [Cascading Style Sheets](https://www.w3.org/TR/CSS2/) [[CSS21]], [Accessible Rich Internet Applications](https://www.w3.org/TR/wai-aria-1.1/) [[WAI-ARIA-1.1]], [Scaleable Vector Graphics](https://www.w3.org/TR/SVG2/) [[SVG2]] and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning that it can conceivably be used to describe test rules for other technologies.
```

Comment:
Please also use this link for WAI-ARIA instead: https://www.w3.org/WAI/intro/aria

Reply:
https://www.w3.org/WAI/intro/aria